### PR TITLE
Dls 4471

### PR DESCRIPTION
**Pull request by ayub96**
[Your pull request description here]

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [ ] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [ ] Requires acceptance test run.
- [ ] Appropriate labels added.
- [ ] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/models/SatisfactionSurvey.scala [hunk 1]
```diff
@@ -64,27 +68,53 @@ object SatisfactionSurvey {
...
+  //      "details" -> details.fold("") { x => x.toString }
+  //    )
+  //  }
+  //
+  //  implicit val jsonWrites_5 =  Writes[SatisfactionSurvey.Fifth] {
+  //    case Fifth(details) => Json.obj(
+  //      "satisfaction" -> "05",
+  //      "details" -> details.fold("") { x => x.toString }
+  //    )
+  //  }
```
Comment (sureshhmrc) [inline app/models/SatisfactionSurvey.scala:118]: you can remove the above if its not needed

File: app/models/businessmatching/BusinessActivities.scala [hunk 2]
```diff
@@ -153,7 +153,7 @@ object BusinessActivities {
   def combinedReader(count: Int, msg: String) = formReaderMinLengthR(msg) andThen maxLengthValidator(count).repath(_ => Path \ "businessActivities")
 
   implicit def activitySetWrites(implicit w: Write[BusinessActivity, String]) = {
-    Logger.info(s"${w.toString}")
+    logger.info(s"${w.toString}")
```
Comment (sureshhmrc) [inline app/models/businessmatching/BusinessActivities.scala:156]: not specific to this PR but these logs are useless , but no need to worry

File: app/connectors/AmlsConnector.scala [hunk 3]
```diff
@@ -46,12 +47,12 @@ class AmlsConnector @Inject()(val http: HttpClient,
     val postUrl = s"$url/$accountType/$accountId/$safeId"
     val prefix = "[AmlsConnector][subscribe]"
     // $COVERAGE-OFF$
-    Logger.debug(s"$prefix - Request Body: ${Json.toJson(subscriptionRequest)}")
+    logger.debug(s"$prefix - Request Body: ${Json.toJson(subscriptionRequest)}")
```
Comment (sureshhmrc) [inline app/connectors/AmlsConnector.scala:50]: not for this PR, but we can remove all the debug logs as these won't be printed anyway in live environment. but leave it for now

File: project/plugins.sbt [hunk 4]
```diff
@@ -5,11 +5,11 @@ resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.
...
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.5.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.24" exclude("org.slf4j", "slf4j-simple"))
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.8.8" exclude("org.slf4j", "slf4j-simple"))
 
 addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.10")
```
Comment (sureshhmrc) [inline project/plugins.sbt:14]: i guess we can remove this `sbt-release` ?

**Comment by sureshhmrc** [comment]
also please make sure there are no warnings during service startup 
i guess https://github.com/hmrc/amls-frontend/blob/master/conf/application.conf#L101-L112 lines produce some warnings ?

**Comment by ayub96** [comment]
> also please make sure there are no warnings during service startup
> i guess https://github.com/hmrc/amls-frontend/blob/master/conf/application.conf#L101-L112 lines produce some warnings ?

I have now removed the lines you highlighted

File: app/models/SatisfactionSurvey.scala (hunk 1 above)
Comment (sureshhmrc) [reply app/models/SatisfactionSurvey.scala]: can you remove these commented lines as well please if its not required ?

**Review by sureshhmrc — APPROVED** [review]

**Pull request merged**